spec/reek/smells/attribute_spec.rb

```diff
@@ -40,210 +43,80 @@
           attr :super_thing2
           private
           attr :super_thing2
-        end"
-      end
-
-      it_should_behave_like 'no attribute found'
+        end
+      ').to_not reek_of(:Attribute)
     end
 
-    context 'declared in a module' do
-      before :each do
-        @src = "module Fred
```

Comment:
Just a single test to make sure this detector also checks modules should be enough. Just enough to justify the implementation of `self.contexts`.